### PR TITLE
Handles empty dict tracks

### PR DESCRIPTION
**Pull request by orilg**
Handle tracks in weird mixes that come back as {} from api

### CODE REVIEW

File: orochi/client.py [hunk 1]
```diff
@@ -528,6 +528,9 @@ def _song_end_handler(self, signum, frame):
             except RuntimeError as e:
                 print('*** RuntimeError: {}'.format(e))
                 self.do_stop()
+            except KeyError as e:
+                print('*** KeyError: {}'.format(e))
```
Comment (dbrgn) [inline orochi/client.py]: Can you maybe instead print something like this?

    print('*** Error: Missing song information in API response. Skipping mix.')

That's probably more useful to the user.

**Comment by orilg** [comment]
sure

**Comment by dbrgn** [comment]
Thanks!

**Comment by dbrgn** [comment]
This was a follow-up to #65.

**Pull request merged**